File: javascript/abstractions-publish-gateway_e5bab6.js
#!/usr/bin/env node
// abstractions-publish-gateway.tcl — portiert nach javascript
// Quelle: tcl, Projects@abstractions:tcl/abstractions-publish-gateway.tcl
// Erzeugt: 2026-08-08 durch ABSTRACTIONS_MANAGER.py

// abstractions-publish-gateway.sh — portiert nach JavaScript
// Quelle: shell, OpenClaw@gateway2:scripts/abstractions-publish-gateway.sh
// Erzeugt: 2026-08-07 durch ABSTRACTIONS_MANAGER.py

// Workspace-visible wrapper for the gateway publish job.

const { spawn } = require('child_process');
const path = require('path');

const scriptPath = path.join('/home/openclaw/.openclaw/scripts/abstractions-publish-gateway.sh');

const child = spawn(scriptPath, process.argv.slice(2), {
    stdio: 'inherit'
});

child.on('error', (err) => {
    console.error(`Fehler beim Ausführen des Skripts: ${err.message}`);
    process.exit(1);
});

child.on('close', (code) => {
    process.exit(code);
});

File: python/abstractions-publish-gateway_e5bab6.py
#!/usr/bin/env python3
# abstractions-publish-gateway.tcl — portiert nach python
# Quelle: tcl, Projects@abstractions:tcl/abstractions-publish-gateway.tcl
# Erzeugt: 2026-08-08 durch ABSTRACTIONS_MANAGER.py

# abstractions-publish-gateway.sh — portiert nach tcl
# Quelle: shell, OpenClaw@gateway2:scripts/abstractions-publish-gateway.sh
# Erzeugt: 2026-08-07 durch ABSTRACTIONS_MANAGER.py

# Workspace-visible wrapper for the gateway publish job.
import subprocess
import sys
import os

# Führe das Shell-Skript mit denselben Argumenten aus
result = subprocess.run([
    '/home/openclaw/.openclaw/scripts/abstractions-publish-gateway.sh'
] + sys.argv[1:])

# Beende das Python-Skript mit demselben Exit-Code wie das aufgerufene Skript
sys.exit(result.returncode)
